prow/jobs/test-infra/test-jobs-yaml-definitions.yaml

```diff
@@ -0,0 +1,20 @@
+presubmits: # runs on PRs
+  kyma-project/kyma:
+  - name: &name prow/test-infra/test-jobs-yaml-definitions
+    run_if_changed: "(/prow/jobs)|(/development/tools/jobs)"
+    context: *name
+    branch: master
+    skip_report: false
+    decorate: true
+    path_alias: github.com/kyma-project/test-infra
+    spec:
+      containers:
+      - image: eu.gcr.io/kyma-project/prow/test-infra/buildpack-golang:v20181119-afd3fbd
+        workingDir: "/home/prow/go/src/github.com/kyma-project/test-infra/development/tools/jobs/"
+        env:
+        - name: GOPATH
```

Comment:
Is it not defined by default? If not, it should be defined in buildpack as decorator always checkout to `home/prow/go/src...`.

Reply:
it should be, I will remove this env definition